samples/interactive-tutorials/product/crud_product.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START retail_crud_product]
+# Create product in a catalog using Retail API
+#
+import os
+
+from google.api_core.client_options import ClientOptions
+
+from google.cloud.retail import CreateProductRequest, DeleteProductRequest, \
+    GetProductRequest, Product, ProductServiceClient, UpdateProductRequest
+from google.cloud.retail_v2 import PriceInfo
+from google.cloud.retail_v2.types import product
+from setup.setup_cleanup import try_to_delete_product_if_exists
+
+project_number = os.getenv('GOOGLE_CLOUD_PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+default_branch_name = "projects/" + project_number + "/locations/global/catalogs/default_catalog/branches/default_branch"
+product_id = 'crud_product_id'
```

Comment:
Prefer we use a random id
''.join(random.sample(string.ascii_lowercase, 8))

So that we can just remove try_to_delete_product_if_exists from the doc, which may confuse users.

Reply:
fixed